src/Core/Content/MeasurementSystem/DataAbstractionLayer/MeasurementUnits.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\MeasurementSystem\DataAbstractionLayer;
+
+use Shopware\Core\Content\MeasurementSystem\Service\ConvertedUnit;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Struct\Struct;
+
+#[Package('inventory')]
+class MeasurementUnits extends Struct
+{
+    /**
+     * @param array<string, ConvertedUnit> $valueByUnit
+     */
+    public function __construct(protected array $valueByUnit = [])
+    {
+    }
+
+    public function add(string $type, ConvertedUnit $unit): void
+    {
+        $this->valueByUnit[$type] = $unit;
+    }
+
+    public function jsonSerialize(): array
+    {
+        return array_map(
+            static fn (ConvertedUnit $unit) => [
```

Comment:
shouldn't this be part of the json serialize in the `ConvertedUnit` class?

Reply:
I want to get rid of `valueByUnit` in the api response so I spread the `valueByUnit` here